pkg/iac-providers/kustomize/commons/types.go

```diff
@@ -0,0 +1,32 @@
+package commons
+
+import (
+	"github.com/accurics/terrascan/pkg/utils"
+	"github.com/hashicorp/go-multierror"
+)
+
+const (
+	// YAMLExtension yaml
+	YAMLExtension = "yaml"
+	// YAMLExtension2 yml
+	YAMLExtension2 = "yml"
+	// KustomizeFileName kustomization
+	KustomizeFileName = "kustomization"
+)
+
+// KustomizeDirectoryLoader implements kustomize directory/file loading
+type KustomizeDirectoryLoader struct {
```

Comment:
if this struct implements both directory and file loading should the type be `KustomizeLoader`?